ml-agents/mlagents/trainers/torch/networks.py

```diff
@@ -40,7 +40,11 @@ def __init__(
             else 0
         )
 
-        self.visual_processors, self.vector_processors, encoder_input_size = ModelUtils.create_input_processors(
```

Comment:
Is this reformatting necessary?

Reply:
Weird, it was done by my `black` but got undid when I ran it again. Removed.